### PR TITLE
Extend the notes about noexec option for mounting /tmp and /var/tmp (#1677)

### PR DESCRIPTION
**Pull request by keichwa**
# Description
Extend the notes about noexec option for mounting /tmp and /var/tmp (#1677)

* Extend the note about noexec mount option
* Add important note to the Actions section
* Put an extra note about noexec to saltbundle section
Co-authored-by: Karl Eichwalder <ke@suse.de>

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)
- [x] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: .changelog [hunk 1]
```diff
@@ -4,6 +4,8 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Extend the notes about using noexec option for /tmp and /var/tmp (bsc#1201210)
+- Reverted single snippet change for two separate books
```
Comment (keichwa) [inline .changelog:8]:
```suggestion
```

File: modules/client-configuration/pages/contact-methods-saltbundle.adoc [hunk 2]
```diff
@@ -88,6 +88,13 @@ The bootstrap repository must be created before bootstrapping the client with {w
 For more information, see xref:client-configuration:bootstrap-repository.adoc#_prepare_to_create_a_bootstrap_repository[Prepare to Create a Bootstrap Repository]
 ====
 
+[IMPORTANT]
+====
+Salt SSH is using [path]``/var/tmp`` to deploy Salt Bundle to and execute Salt commands on the client with the bundled Python.
```
Comment (0rnela) [inline modules/client-configuration/pages/contact-methods-saltbundle.adoc:93]:
```suggestion
Salt SSH is using [path]``/var/tmp`` to deploy Salt Bundle and to execute Salt commands on the client with the bundled Python.
```

File: modules/installation-and-upgrade/pages/client-requirements.adoc [hunk 3]
```diff
@@ -275,6 +275,13 @@ ifeval::[{uyuni-content} == true]
 |===
 endif::[]
 
+[IMPORTANT]
+====
+Salt SSH is using [path]``/var/tmp`` to deploy Salt Bundle to and execute Salt commands on the client with the bundled Python.
```
Comment (0rnela) [inline modules/installation-and-upgrade/pages/client-requirements.adoc:280]:
```suggestion
Salt SSH is using [path]``/var/tmp`` to deploy Salt Bundle and to execute Salt commands on the client with the bundled Python.
```

**Review by 0rnela — APPROVED** [review]
Please add the suggested corrections before the merge.

File: modules/client-configuration/pages/contact-methods-saltbundle.adoc (hunk 2 above)
Comment (keichwa) [reply modules/client-configuration/pages/contact-methods-saltbundle.adoc:93]: https://github.com/uyuni-project/uyuni-docs/pull/1693#discussion_r934252387 also applies here.

**Pull request merged**